### PR TITLE
Upgrade

### PR DESCRIPTION
**Pull request by edewit**
tested it against the unit tests, but couldn't get the example project to work the oauth service needs to be handled with a continuation manager.

### CODE REVIEW

**Comment by wei-lee** [comment]
looks good to me. I merged PR #4 to master already. Do you want to merge master back into this branch and use unity for the DI?

**Comment by edewit** [comment]
rebased @wei-lee

File: FHSDKPhone81/FHClient.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    /// 
+    /// </code>
+    /// </example>
+    /// <returns>If Init is success or not</returns>
+    /// <exception cref="FHException"></exception>
+    public class FHClient : FH
+    {
+        public new static async Task<bool> Init()
+        {
+            return await FH.Init();
```
Comment (wei-lee) [inline FHSDKPhone81/FHClient.cs]: do you need to inject the dependencies like [this](https://github.com/feedhenry/fh-dotnet-sdk/blob/master/FHSDKPhone/FHClient.cs#L59)?

**Comment by edewit** [comment]
right updated, still this a bit of duplication, but I don't have a better idea atm

File: FHSDKPhone81/Services/Auth/OAuthClientHandlerService.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            : base()
+        {
+
+        }
+
+        public override Task<OAuthResult> Login(string oauthLoginUrl)
+        {
+            tcs = new TaskCompletionSource<OAuthResult>();
+            Uri uri = new Uri(oauthLoginUrl, UriKind.Absolute);
+            var oauth = await WebAuthenticationBroker.AuthenticateAsync(WebAuthenticationOptions.None, uri);
```
Comment (wei-lee) [inline FHSDKPhone81/Services/Auth/OAuthClientHandlerService.cs]: I am getting the following compiler error here: 
The 'await' operator can only be used within an async method. Consider marking this method with the 'async' modifier and changing its return type to 'Task<System.Threading.Tasks.Task<FHSDK.Services.OAuthResult>>'.	C:\Users\FeedHenry\Documents\GitHub\fh-dotnet-sdk\FHSDKPhone81\Services\Auth\OAuthClientHandlerService.cs.

**Comment by wei-lee** [comment]
Another thing is is there anyway to test for wp8.1? We have a test project for wp8, but I didn't see there is one for wp8.1. Might be a good idea to create one too? or should we retarget the existing one to wp8.1?

**Pull request merged**